ironfish/src/rpc/routes/router.ts

```diff
@@ -52,7 +52,15 @@ export class Router {
       throw new RouteNotFoundError(route, namespace, method)
     }
 
-    const { handler, schema } = methodRoute
+    const { handler, schema, experimental } = methodRoute
+
+    if (experimental && !this.server.context.config.experimental) {
```

Comment:
I would prefer not to add a blanket feature like this. It should be feature based. I also don't think we need to gate anything with this right now, do we? What would we need to gate?

Reply:
We don't necessarily need it right now, but we do have RPCs exposed that people technically shouldn't be using. It could be helpful in this case, lets say if a bridge integrator starts operating with this before we complete the feature.


This PR is not fully baked at all, just using as a placeholder for discussion.

Reply:
FWIW I agree, namespacing the features seems like a good idea.